src/services/cooperation-service.ts

```diff
@@ -62,44 +61,66 @@ export const cooperationService = {
         parameters: { id }
       })
     })
+  },
+  updateResourceCompletionStatus: ({
+    completionStatus,
+    id,
+    resourceId
+  }: UpdateResourceCompletionStatusParams) => {
+    return baseService.request<void>({
+      data: { completionStatus },
+      method: 'PATCH',
+      url: getFullUrl({
+        pathname: URLs.cooperations.updateStatusById,
+        parameters: { id, resourceId }
+      })
+    })
   }
 }
 
 export const CooperationNotesService = {
-  getNotes: async (cooperationId?: string): Promise<AxiosResponse> =>
-    await axiosClient.get(
-      createUrlPath(
-        `${URLs.cooperations.get}/${cooperationId}${URLs.notes.get}`
-      )
-    ),
-  createNote: async (
-    data?: CreateOrUpdateNoteParams,
-    cooperationId?: string
-  ): Promise<AxiosResponse> =>
-    await axiosClient.post(
-      createUrlPath(
-        `${URLs.cooperations.get}/${cooperationId}${URLs.notes.create}`
-      ),
+  getNotes: (cooperationId: string): Promise<AxiosResponse> => {
```

Comment:
You forget to add generic to the `AxiosResponse`
```suggestion
  getNotes: (cooperationId: string): Promise<AxiosResponse<NoteResponse[]>> => {
```

Reply:
done